integreat_cms/cms/forms/events/event_form.py

```diff
@@ -171,12 +171,12 @@ def clean(self):
                                 code="invalid",
                             ),
                         )
-            elif cleaned_data["end_date"] - cleaned_data["start_date"] > timedelta(6):
+            elif cleaned_data["end_date"] - cleaned_data["start_date"] > timedelta(27):
                 self.add_error(
                     "end_date",
                     forms.ValidationError(
                         _(
-                            "The maximum duration for events is 7 days. Consider using recurring events if the event is not continuous."
+                            "The maximum duration for events is 28 days. Consider using recurring events if the event is not continuous."
```

Comment:
To me, the new maximum duration also sounds a bit random, maybe we should add a new setting for this to be able to change it a bit easier in the future?

Reply:
A nice idea 😃 I added it to `settings`.

We can also implement in such a way that service team can change it by themselvs, but it's an over kill in my opinion 😅

Reply:
Indeed, no need to store this in the database (yet :laughing:)